plugins/org.eclipse.glsp.api/src/org/eclipse/glsp/api/types/Item.java

```diff
@@ -0,0 +1,48 @@
+/********************************************************************************
+ * Copyright (c) 2020 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+package org.eclipse.glsp.api.types;
+
+import java.util.List;
+
+import org.eclipse.glsp.api.action.Action;
+
+public class Item extends LabeledAction {
```

Comment:
Maybe we find a better name than `Item`, it feels to be too generic.
E.g. we could call it `MenuItem` and rename the current menu item into `ContextMenuItem`?